Lib/venv/__init__.py

```diff
@@ -409,8 +429,52 @@ def create(env_dir, system_site_packages=False, clear=False,
                          prompt=prompt, upgrade_deps=upgrade_deps)
     builder.create(env_dir)
 
+
+def get_default_args(venv_ini_path=VENV_INI_DEFAULT_PATH):
+    """ Check if a venv.ini exists in the platform config location
+        + set defaults if conf exists. Use `--help` to test ini config """
+    bool_keys = {
+        "clear", "copies", "system_site_packages", "upgrade_deps",
+        "use_symlinks", "without_pip"
+    }
```

Comment:
To make sure this stays current:
```py3
bool_keys = {
    key
    for key, value_type in Defaults._field_types.items()
    if value_type is bool
}
```

Reply:
I did discuss above this option above. Cool, if you think it's not crazy I'll do it too.